test/config/fileserver_config.json

```diff
@@ -0,0 +1,41 @@
+{"os_image":

```

Comment:
These configuration files contain lab specific information and need to move into the internal repository, we can work on that with you.